TO-DO-APP/app/src/main/java/com/example/to_do_app/Adapter.kt

```diff
@@ -1,44 +0,0 @@
-package com.example.to_do_app
-
-import android.content.Intent
-import android.graphics.Color
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import androidx.recyclerview.widget.RecyclerView
-import kotlinx.android.synthetic.main.view.view.*
-
-class Adapter(var data: List<CardInfo>) : RecyclerView.Adapter<Adapter.viewHolder>() {
-    class viewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-        var title = itemView.title
-        var priority = itemView.priority
-        var layout = itemView.mylayout
-    }
-
-    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): viewHolder {
-        var itemView = LayoutInflater.from(parent.context).inflate(R.layout.view, parent, false)
-        return viewHolder(itemView)
-    }
-
-    override fun onBindViewHolder(holder: viewHolder, position: Int) {
-        when (data[position].priority.toLowerCase()) {
```

Comment:
converting string to lowerCase you could have created enums or StringDefs for this,